doc/examples/plot_restoration.py

```diff
@@ -4,53 +4,121 @@
 Image Deconvolution
 =====================
 
-In this example, we deconvolve a noisy version of an image using Wiener
-and unsupervised Wiener algorithms. This algorithms are based on
-linear models that can't restore sharp edge as much as non-linear
-methods (like TV restoration) but are much faster.
-
-Wiener filter
--------------
-The inverse filter based on the PSF (Point Spread Function),
-the prior regularisation (penalisation of high frequency) and the
+In this example, we deconvolve a noisy version of an image using
+Richardson–Lucy, Wiener, and unsupervised Wiener deconvolution algorithms.
+These algorithms are based on linear models that can't restore sharp edge as
+much as non-linear methods (like TV restoration) but are much faster.
+
+Richardson–Lucy deconvolution
+-------------------
```

Comment:
Extend "-" to the full width of the heading, also below for Wiener deconvolution.

Thanks.